ecommerce/api.py

```diff
@@ -96,8 +97,10 @@ def get_purchasable_course_run(course_key, user):
             order__user=user,
             course_key=course_run.edx_course_key,
```

Comment:
is this enough to figure out that user has paid for course?
i think these filters are unique for edx_course_key
```python
order__status=Order.FULFILLED,
order__user=user,
 course_key=course_run.edx_course_key,
```


```python
if Line.objects.filter(
            order__status=Order.FULFILLED,
            order__user=user,
            course_key=course_run.edx_course_key,
    ).exists():
```